lib/parser/youtube/VideoInfo.ts

```diff
@@ -0,0 +1,414 @@
+                has_ypc_metadata: info.microformat?.has_ypc_metadata
+            },
+            like_count: undefined as number | undefined,
+            is_liked: undefined as boolean | undefined,
+            is_disliked: undefined as boolean | undefined,
+        };
+        this.streaming_data = info.streaming_data;
+        this.playability_status = info.playability_status;
+        this.annotations = info.annotations;
+        this.storyboards = info.storyboards;
+        this.endscreen = info.endscreen;
+        this.captions = info.captions;
+        this.cards = info.cards;
+        const two_col = Parser.cast_response(next.contents, TwoColumnWatchNextResults, false);
+        const results = two_col?.results;
+        const secondary_results = two_col?.secondary_results;
+        if (results && secondary_results) {
+            this.primary_info = Parser.cast_response(results.get({ type: 'VideoPrimaryInfo' }), VideoPrimaryInfo, false);
+            this.secondary_info = Parser.cast_response(results.get({ type: 'VideoSecondaryInfo' }), VideoSecondaryInfo, false);
+            this.merchandise = Parser.cast_response(results?.get({ type: 'MerchandiseShelf' }), MerchandiseShelf, false);
+            this.related_chip_cloud = Parser.cast_response(
+                Parser.cast_response(
+                    secondary_results?.get({ type: 'RelatedChipCloud' }), 
+                    RelatedChipCloud, 
+                    false)?.content, 
+                ChipCloud, 
+                false
+            );
+            this.watch_next_feed = Parser.cast_response(secondary_results?.get({ type: 'ItemSection' }), ItemSection, false)?.contents;
+            if (this.watch_next_feed && Array.isArray(this.watch_next_feed)) 
+                this.#watch_next_continuation = Parser.cast_response(this.watch_next_feed?.pop(), ContinuationItem, false);
+            this.player_overlays = Parser.cast_response(next.player_overlays, PlayerOverlay);
+            this.basic_info.like_count = Parser.cast_response(this.primary_info?.menu?.top_level_buttons?.get({ icon_type: 'LIKE' }), ToggleButton, false)?.like_count;
+            this.basic_info.is_liked = Parser.cast_response(this.primary_info?.menu?.top_level_buttons?.get({ icon_type: 'LIKE' }), ToggleButton, false)?.is_toggled;
+            this.basic_info.is_disliked = Parser.cast_response(this.primary_info?.menu?.top_level_buttons?.get({ icon_type: 'DISLIKE' }), ToggleButton, false)?.is_toggled;
+            const comments_entry_point = Parser.cast_response(results.get({ target_id: 'comments-entry-point' }), ItemSection, false);
+            this.comments_entry_point_header = Parser.cast_response(comments_entry_point?.contents?.get({ type: 'CommentsEntryPointHeader' }), CommentsEntryPointHeader, false);
+            /**
+             * @type {import('../classes/LiveChat')}
+             */
+            // TODO: this makes no sense, LiveChat is not in the parsers list
```

Comment:
I'm a bit confused — what happened here?

Reply:
There's no LiveChat parser in classes, the memo can thus never contain it. There's a LiveChat class in lib/parser/youtube/LiveChat though but not sure how you want to instantiate it.

Reply:
> There's no LiveChat parser in classes, the memo can thus never contain it. 

[There actually is](https://github.com/LuanRT/YouTube.js/blob/main/lib/parser/classes/LiveChat.js), maybe it got accidentally deleted?

Reply:
Oh must have missed it when I wrote that, I'll fix that in next push!